storage/queued_country.cpp

```diff
@@ -45,6 +45,6 @@ bool QueuedCountry::SwitchToNextFile()
          ("Current file (", m_current, ") is not specified in left files (", m_left, ")."));
   m_left = UnsetOptions(m_left, m_current);
   m_current = LeastSignificantOption(m_left);
-  return m_current != MapOptions::Nothing;
+  return m_current != MapOptions::Nothing && m_current != MapOptions::CarRouting;
```

Comment:
Why you updated the condition? It looks like this change breaks logic for large mwms with separate map and routing files, right? Moreover, even for small mwms it should not be possible to select CarRouting from downloader's menu.

To sum up, do we really need this change?